bindings/python/pinocchio/__init__.py

```diff
@@ -3,6 +3,12 @@
 #
 
 import numpy as np
+
+import sys
+if 'eigenpy' not in sys.modules:
+    import eigenpy
+    eigenpy.switchToNumpyMatrix() # forced default conversion towards. Silent the warning since EigenPy >= 1.6.0
```

Comment:
Is Pinocchio compatible with both matrices and arrays ?
If yes, I think you shouldn't make the warning silent in a module. The warning should reach the users otherwise they will not be warned about the EigenPy future changes.

Reply:
Not the tutorials and RobotWrapper if I remember well.